thinc/layers/dropout.py

```diff
@@ -1,71 +1,119 @@
-from typing import Tuple, Callable, List, TypeVar, Any
+from typing import Tuple, Callable, List, TypeVar, cast, Union
 
 from ..model import Model
 from ..config import registry
-from ..types import ArrayXd, Ragged, Padded
-
-
-InT = TypeVar("InT")
-ArrayT = TypeVar("ArrayT", bound=ArrayXd)
+from ..types import (
+    ArrayXd,
+    Ragged,
+    Padded,
+    Ints1d,
+    Ints2d,
+    Ints3d,
+    Ints4d,
+    Floats1d,
+    Floats2d,
+    Floats3d,
+    Floats4d,
+    FloatsXd,
+    IntsXd,
+)
+
+
```

Comment:
Here I had to list all the individual types of list: a solution with a covariant type variable wasn't possible because covariant type variables can't be nested.

Reply:
In all these cases where the `import` spans multiple lines, can you rewrite them into separate `import` statements instead?